config/settings.yml

```diff
@@ -17,6 +17,13 @@
     :password:
     :port:
     :user:
+  :gce:
+    :host:
+    :password:
+    :port:
+    :user:
+:log:
+  :level_gce: info
```

Comment:
Also, this is gce here (and in the core logger setup), but I feel like it should be `level_google` in a future PR.

Reply:
Were/are there multiple google cloud providers maybe? Something like scvmm vs azure?

Reply:
Yeah with the growing number of ems_types per provider plugin we should look at what use ems_type vs provider